ttnn/cpp/ttnn/operations/embedding/embedding_pybind.hpp

```diff
@@ -40,7 +40,7 @@ void py_module(py::module& module) {
 
 
         Returns:
-            ttnn.Tensor: the output tensor.
+            ttnn.Tensor: the output tensor of layout == layout or layout of the weights tensor.
```

Comment:
consider this rewording or smth like that
```suggestion
            ttnn.Tensor: the output tensor of a specified layout or with a layout of the weights tensor.
```